daemon/api.go

```diff
@@ -203,6 +204,11 @@ var (
 		POST: postCreateUser,
 	}
 
+	removeUserCmd = &Command{
+		Path: "/v2/remove-user",
```

Comment:
two remarks

* we are trying as we change things to split api.go that is becoming very big into smaller `api_TOPIC.go` and `api_TOPIC_test.go` files, it would be good to do that for user support, maybe in a follow up. There are some examples split out already

* I have ambivalent feelings about adding /remove-user , already /create-user doesn't quite fit into our patterns.  The other solution would be to add removal as a mode to /create-user,  a bit strange but less non-fitting API surface. Maybe @chipaca has input/opinions on this